portfolio/src/main/java/com/google/sps/classes/UtilityClass.java

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.classes;
+
+import com.google.gson.Gson;
+import com.google.sps.classes.UserAuthentication;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * User object created to group login/logout url and status.
+ */
```

Comment:
Typically for utility classes, you'll want them to be a bit more generic (not specifically for the login/logout URL and status). Consider rewording this to be something like "Utility methods shared across classes."

Reply:
Done!